pandas/core/frame.py

```diff
@@ -77,6 +77,7 @@
     ABCMultiIndex,
     ABCSeries,
 )
+from pandas.core.dtypes.inference import is_hashable
```

Comment:
should be part of the: pandas.core.dtypes.common import above

Reply:
fixed+green